src/functionaltests/java/com/ericsson/ei/restendpoints/RestEndpointsTestSteps.java

```diff
@@ -0,0 +1,121 @@
+package com.ericsson.ei.restendpoints;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Ignore;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.TestPropertySource;
+
+import com.ericsson.ei.utils.FunctionalTestBase;
+import com.ericsson.ei.utils.HttpRequest;
+import com.ericsson.ei.utils.HttpRequest.HttpMethod;
+import com.ericsson.eiffelcommons.helpers.MediaType;
+import com.ericsson.eiffelcommons.subscriptionobject.RestPostSubscriptionObject;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+
+@Ignore
+@TestPropertySource(properties = {
+        "spring.data.mongodb.database: RestEndpointsTestSteps",
+        "missedNotificationDataBaseName: RestEndpointsTestSteps-missedNotifications",
+        "rabbitmq.exchange.name: RestEndpointsTestSteps-exchange",
+        "rabbitmq.consumerName: RestEndpointsTestStepsConsumer" })
+@AutoConfigureMockMvc
+public class RestEndpointsTestSteps extends FunctionalTestBase {
+
+    @LocalServerPort
+    private int applicationPort;
+    private ResponseEntity response;
+
+    @Value("${spring.data.mongodb.database}")
+    private String eiDatabaseName;
+
+    @Value("${aggregated.collection.name}")
+    private String aggrCollectionName;
+
+    @Value("${missedNotificationCollectionName}")
+    private String missedNotificationCollectionName;
+
+    @Value("${missedNotificationDataBaseName}")
+    private String missedNotificationDatabaseName;
+
+    HttpRequest request;
+
+    @Given("^A GET request is prepared$")
+    public void a_GET_request_is_prepared() {
+        request = new HttpRequest(HttpMethod.GET);
+        setRequestDefaults();
+    }
+
+    @Given("^A POST request is prepared$")
+    public void a_POST_request_is_prepared() {
+        request = new HttpRequest(HttpMethod.POST);
+        setRequestDefaults();
+    }
+
+    @Given("^A PUT request is prepared$")
+    public void a_PUT_request_is_prepared() {
+        request = new HttpRequest(HttpMethod.PUT);
+        setRequestDefaults();
+    }
+
+    @Given("^A DELETE request is prepared$")
+    public void a_DELETE_request_is_prepared() {
+        request = new HttpRequest(HttpMethod.DELETE);
+        setRequestDefaults();
+    }
+
+    @Given("^Subscription with name \"([^\"]*)\" is added as body$")
+    public void subscription_with_name_is_added_as_body(String name) throws Throwable {
```

Comment:
i would change the "name" to "subscriptionName" to reflect more what it is

Reply:
Done.